week3/Makefile

```diff
@@ -0,0 +1,16 @@
+test:
+	 cd ml_decision_tree_sample/unit_tests/;pytest
+test_code:
+	cd ml_decision_tree_sample/unit_tests/; pytest test_code.py
+test_model:
+	cd ml_decision_tree_sample/unit_tests/; pytest test_model.py
+test_data:
+	cd ml_decision_tree_sample/unit_tests/; pytest test_data.py
+train:
+	cd ml_decision_tree_sample/; python3 train.py
+model_card_generator:
+	cd ml_decision_tree_sample/; python3 python3 generator_model_card.py
```

Comment:
why do you use `cd`? could you just run 

`pytest .ml_decision_tree_sample/unit_tests/test_code.py' and so on